app-backend/datamodel/src/main/scala/com/azavea/rf/datamodel/Project.scala

```diff
@@ -19,6 +21,9 @@ case class Project(
   description: String,
   visibility: Visibility,
   tileVisibility: Visibility,
+  isAOIProject: Boolean,
+  aoiCadence: Long, /* Milliseconds */
```

Comment:
I'm not a fan of them being in `Option`. That allows for the creation of illegal states, like where `isAOIProject` is `true` but `cadence` is `None`. We can trivially rule this out by being firmer on the types.